build/charts/antrea/values.yaml

```diff
@@ -294,6 +294,18 @@ agent:
   # resources socket is mounted into antrea-agent Pods. Setting it to an empty
   # value disables the mounting.
   kubeletRootDir: "/var/lib/kubelet"
+  antreaSysctlInit:
+    # -- Enable the sysctl override. When enabled, the init container will do the process.
+    enable: true
```

Comment:
Is it enabled by default intentionally? I thought it's only needed on some platforms?